cdk-lib/viewer-stacks/viewer-vpc-stack.ts

```diff
@@ -0,0 +1,85 @@
+import { Construct } from 'constructs';
+import { RemovalPolicy } from 'aws-cdk-lib';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as logs from 'aws-cdk-lib/aws-logs';
+import { Stack, StackProps } from 'aws-cdk-lib';
+
+import * as types from '../core/context-types';
+
+export interface ViewerVpcStackProps extends StackProps {
+    readonly captureTgw: ec2.CfnTransitGateway;
+    readonly captureVpc: ec2.Vpc;
+    readonly viewerVpcPlan: types.VpcPlan;
+}
+
+export class ViewerVpcStack extends Stack {
+  public readonly vpc: ec2.Vpc;
+  public readonly flowLog: ec2.FlowLog;
+
+  constructor(scope: Construct, id: string, props: ViewerVpcStackProps) {
+    super(scope, id, props);
+
+    // The VPC the Viewer Nodes will live in
+    this.vpc = new ec2.Vpc(this, 'VPC', {
```

Comment:
Should we label this ViewerVPC

Reply:
It would be redundant; take a look at the screenshots in the overview.  The CDK prepends the stack name on the VPC name, so the full name is like: `MyCluster-ViewerVPC/VPC`